images/images.qrc

```diff
@@ -305,6 +315,16 @@
         <file>themes/qfield/xhdpi/ic_ring_tool_white_24dp.png</file>
         <file>themes/qfield/xxhdpi/ic_ring_tool_white_24dp.png</file>
         <file>themes/qfield/xxxhdpi/ic_ring_tool_white_24dp.png</file>
+        <file>themes/qfield/hdpi/ic_view_black_48dp.png</file>
+        <file>themes/qfield/mdpi/ic_view_black_48dp.png</file>
+        <file>themes/qfield/xhdpi/ic_view_black_48dp.png</file>
+        <file>themes/qfield/xxhdpi/ic_view_black_48dp.png</file>
+        <file>themes/qfield/xxxhdpi/ic_view_black_48dp.png</file>
```

Comment:
Remove if not needed.

I think it's always green now. 
But it could be black in edit mode still. I have no strong opinion about.